dev/tests/functional/tests/app/Magento/Customer/Test/Repository/Address.xml

```diff
@@ -118,6 +118,21 @@
             <field name="default_shipping" xsi:type="string">Yes</field>
         </dataset>
 
+        <dataset name="US_address_TX_Wrong_Validation">
```

Comment:
I am wondering why did you created MFTF test but added the dataset into MTF (old one)?

Reply:
I thought that my test has similar steps as old one(for example create order) and thats why  I based in previously data and only add invalid first name.

Reply:
I saw that. But my question was about adding a new MFTF test and referring to newly created dataset in MFT test. I mean that MTF and MFTF are completely different testing frameworks